src/LangOpts.zig

```diff
@@ -44,28 +44,25 @@ const Standard = enum {
     pub fn atLeast(self: Standard, other: Standard) bool {
         return @enumToInt(self) >= @enumToInt(other);
     }
+
+    pub fn isGNU(standard: Standard) bool {
+        return switch (standard) {
+            .gnu89, .gnu99, .gnu11, .gnu17, .gnu2x => true,
+            else => false,
+        };
+    }
 };
 
 standard: Standard = .gnu17,
 
-pub fn hasGNUKeywords(langopts: LangOpts) bool {
-    return switch (langopts.standard) {
-        .gnu89, .gnu99, .gnu11, .gnu17, .gnu2x => true,
-        else => false,
-    };
-}
-
-pub fn hasC99Keywords(langopts: LangOpts) bool {
-    return langopts.standard.atLeast(.c99);
-}
-
 pub fn setStandard(self: *LangOpts, name: []const u8) error{InvalidStandard}!void {
     self.standard = Standard.NameMap.get(name) orelse return error.InvalidStandard;
 }
 
 pub fn suppress(langopts: LangOpts, tag: DiagnosticTag) bool {
     return switch (tag) {
         .static_assert_missing_message => langopts.standard.atLeast(.c2x),
+        .alignof_expr => langopts.standard.isGNU(),
```

Comment:
Clang always shows this error and gcc never does, should we maybe distinguish between `gnu17` and `default` and allow gnu extensions with `default` but show errors if a gnu version was not explicitly specified?